lang/strings_polish.txt

```diff
@@ -726,6 +726,7 @@ $s_reset_password_button = 'Nowe hasło';
 $s_delete_user_button = 'Usuń Użytkownika';
```

Comment:
This is not the way we change on English messages.
Translations are managed via TranslateWiki. Please refer our own Wiki [1] for the process to submit new translations or revise existing ones.

[1] http://www.mantisbt.org/wiki/doku.php/mantisbt:translationshowto